java-vertexai/google-cloud-vertexai/pom.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud</groupId>
+  <artifactId>google-cloud-vertexai</artifactId>
+  <version>0.0.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-vertexai:current} -->
+  <packaging>jar</packaging>
+  <name>Google VertexAI API</name>
+  <description>VertexAI API Vertex AI is an integrated suite of machine learning tools and services for building and using ML models with AutoML or custom code. It offers both novices and experts the best workbench for the entire machine learning development lifecycle.</description>
+  <parent>
+    <groupId>com.google.cloud</groupId>
+    <artifactId>google-cloud-vertexai-parent</artifactId>
+    <version>0.0.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-vertexai:current} -->
+  </parent>
+  <properties>
+    <site.installationModule>google-cloud-vertexai</site.installationModule>
+    <mockito.version>5.7.0</mockito.version>
```

Comment:
I believe the mockito version is already managed.

Reply:
Changed my dependency to mockito-all without version specification. Thanks!